### PR TITLE
Added EPD metrics to homepage

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
image

### CODE REVIEW

File: frontend/components/EdpMetrics.vue [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<template>
```
Comment (AndreiNavoichyk) [inline frontend/components/EdpMetrics.vue]: Rename to EDPBanner

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 vNext
 - Added API to add and get EDP info. 
+- Added UI to show EDP metrics to homepage.
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Rename to EDP banner

File: frontend/components/EdpMetrics.vue (hunk 1 above)
Comment (karthikeyan-cere) [reply frontend/components/EdpMetrics.vue]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md]: Updated

File: api/postman/Cerestats.postman_collection.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "{\n    \"key\" : \"test1\",\n    \"value\": {\n        \"graduates-number\": \"10\"\n    }\n}",
+					"raw": "{\n    \"key\" : \"metrics\",\n    \"value\": {\n        \"show\": \"true\",\n        \"feedbackPercentage\": \"90\",\n        \"onClickLink\": \"http://cere.network\",\n        \"cereBootcampGraduatesNumber\": \"67\",\n        \"tokensRewarded\": \"95000\"\n    }\n}",
```
Comment (AndreiNavoichyk) [inline api/postman/Cerestats.postman_collection.json]: Update with `api/v1/edp/banner`

File: frontend/components/EDPBanner.vue [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  },
+  computed: {
+    formatNumber() {
+      const formatter = Intl.NumberFormat('en', { notation: 'compact' })
+      return formatter.format(this.tokenRewarded)
+    },
+  },
+  apollo: {
+    $subscribe: {
+      metrics: {
```
Comment (AndreiNavoichyk) [inline frontend/components/EDPBanner.vue]: banner

File: api/postman/Cerestats.postman_collection.json (hunk 3 above)
Comment (karthikeyan-cere) [reply api/postman/Cerestats.postman_collection.json]: Updated

File: frontend/components/EDPBanner.vue (hunk 4 above)
Comment (karthikeyan-cere) [reply frontend/components/EDPBanner.vue]: Updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**